ports/pcapplusplus/vcpkg.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "pcapplusplus",
+  "version-string": "21.05",
```

```suggestion
  "version": "21.05",
```
[version ](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version) might be better to be served here. Sorry, I didn't notice this point before.